### PR TITLE
fix(user mgmt concepts): clarifying type vs roles

### PR DESCRIPTION
**Pull request by zuluecho9**
More work based on changes made in https://github.com/newrelic/docs-website/pull/11652/files and https://github.com/newrelic/docs-website/pull/11648/files

Basically: we previously described user type as taking 'precedence' over roles, but this isn't true; both roles and user type work in concert to determine permissions. You can be restricted by either one, in practice.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 1]
```diff
@@ -26,12 +26,18 @@ Here we'll explain how New Relic users get access to specific capabilities and s
...
 
-Let's say a basic user is assigned a group with a role with wide New Relic access, like [**All product admin**](#standard-roles). That user's user type (basic user) would prevent them from using some of the features that a core user or full platform user would have access to. A User must have access to a feature from both their user type and their access control configuration.
+### How user type and role access intersect [#user-type-role]
 
-This doc is focused on role-related permissions. For details about user types, see [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type).
+A New Relic user is given permission to use a New Relic feature by the combination of their a) user type, and their b) role permissions. For a New Relic user to access something, that must be allowed by both the user type and an assigned role.
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:34]:
```suggestion
A New Relic user is given permission to use a New Relic feature by the combination of their (a) user type, and their (b) role permissions. For a New Relic user to access something, they need both the correct user type and assigned role.  
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 2]
```diff
@@ -26,12 +26,18 @@ Here we'll explain how New Relic users get access to specific capabilities and s
...
+### How user type and role access intersect [#user-type-role]
 
-This doc is focused on role-related permissions. For details about user types, see [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type).
+A New Relic user is given permission to use a New Relic feature by the combination of their a) user type, and their b) role permissions. For a New Relic user to access something, that must be allowed by both the user type and an assigned role.  
+
+For example, let's say a basic user is assigned a role with a lot of New Relic access, like [**All product admin**](#standard-roles). Their user type (basic user) would prevent them from using many of the features that a core user or full platform user can access. In order to get more access, they'd have to upgrade to a core or full platform user.
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:36]:
```suggestion
For example, let's say a basic user is assigned a role with a lot of New Relic access, like [**All product admin**](#standard-roles). Their user type (basic user) would prevent them from using many of the features that a core user or full-platform user can access. In order to get more access, they'd have to upgrade to a core or full-platform user. 
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 3]
```diff
@@ -26,12 +26,18 @@ Here we'll explain how New Relic users get access to specific capabilities and s
...
 
-This doc is focused on role-related permissions. For details about user types, see [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type).
+A New Relic user is given permission to use a New Relic feature by the combination of their a) user type, and their b) role permissions. For a New Relic user to access something, that must be allowed by both the user type and an assigned role.  
+
+For example, let's say a basic user is assigned a role with a lot of New Relic access, like [**All product admin**](#standard-roles). Their user type (basic user) would prevent them from using many of the features that a core user or full platform user can access. In order to get more access, they'd have to upgrade to a core or full platform user. 
+
+As another example: let's say a full platform user has a restrictive role assigned (like [**Read only**](#standard-roles)). A full platform user can theoretically access all of New Relic, but in this case their assigned role would restrict their access. To get more access, their assigned roles would need to be changed (for example, by assigning them to a different group, or adjusting the roles assigned to their group).
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:38]:
```suggestion
As another example, let's say a full-platform user has a restrictive role assigned (like [**Read only**](#standard-roles)). A full-platform user can theoretically access all of New Relic, but in this case their assigned role would restrict their access. To get more access, their assigned roles would need to be changed (for example, by assigning them to a different group, or adjusting the roles assigned to their group).
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx [hunk 4]
```diff
@@ -24,7 +24,7 @@ Want to learn about how users are calculated for billing purposes? See [New Reli
 
 ## What's the "user type"? [#user-type-defined]
 
-A user's **user type** is what determines the set of New Relic capabilities a user is entitled to access. A user can only access functionality that is both available to their user type and configured in their access control configuration. ([Learn more about factors that affect user access.](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#user-type-groups-relation))
+A user's **user type** is what determines the maximum set of New Relic capabilities a user is entitled to access. (In addition to user type, there can be other role-related restrictions. [Learn more about how user type relates to roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#user-type-groups-relation).)
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx:27]:
```suggestion
A user's **user type** is what determines the maximum set of New Relic capabilities a user is entitled to access. In addition to user type, there can be other role-related restrictions. [Learn more about how user type relates to roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#user-type-groups-relation).
```

**Review by ally-sassman — DISMISSED** [review]
I left some suggestions, but LGTM!

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:36]: we just say 'full platform user' fwiw

**Comment by zuluecho9** [comment]
thanks @ally-sassman

**Pull request merged**